### ISSUE TITLE
Screen reader users should be presented with form errors such that it's clear what fields need attention and easy to find those fields

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1133]
An element with the `role="alert"` will be immediately read by a screen reader if it's visible.  Therefore the most practical approach here might be to have a screen reader only errors block with `role="alert"` that gets populated with errors and made visible when the form submits (should there be any).  This errors block could then be reused/updated after every subsequent form submission. 

Alternatively it may be cleaner to use the aria-live="polite" on inline errors to have them read in sequence as they are added to the page.  However aria live regions have mixed support across browsers/screen readers so this probably should not be the first choice as a solution.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role
https://www.w3.org/WAI/GL/wiki/Using_ARIA_role_of_alert_for_Error_Feedback_in_Forms
